### DISCUSSION TITLE
Integration with Turso and Edge-deployments

### DISCUSSION THREAD

**Discussion by Mouwrice** [shuttle-hq/shuttle#1292] [Q&A] [answered]
Hi,

I have recently been setting up a project using Shuttle + Turso.
As you know, Turso offers edge databases, so then I was wondering if there are any plans for Shuttle to be able to deploy multiple instances of my web api at different locations closer to the Turso databases?

Right now, Shuttle deploys only in the Western-Europe region https://docs.shuttle.rs/introduction/how-shuttle-works#deploying-your-project which is also where my Turso database resides. But what If I were to expand that infrastructure and also have databases running in Australia or US?

**Comment by oddgrd** [comment, marked as answer]
Hey! Yes, we are planning on supporting multiple regions, but I can't give you an ETA for this at the moment. We are working on a public roadmap, however, which should make the timeline clearer.
